core/types/backwards_compat.libevm_test.go

```diff
@@ -86,8 +89,10 @@ func TestBodyRLPBackwardsCompatibility(t *testing.T) {
 			t.Run("Decode", func(t *testing.T) {
 				got := new(Body)
 				err := rlp.DecodeBytes(wantRLP, got)
-				require.NoErrorf(t, err, "rlp.DecodeBytes(rlp.EncodeToBytes(%T), %T) resulted in %s",
-					(*withoutMethods)(body), got, pretty.Sprint(got))
+				require.NoErrorf(
+					t, err, "rlp.DecodeBytes(rlp.EncodeToBytes(%T), %T) resulted in %s",
+					(*withoutMethods)(body), got, pretty.Sprint(got),
+				)
```

Comment:
nit call me a git maniac but this is formatting is a bit out of scope (spent a solid 30 seconds trying to find what changed 😄) - I would revert (although I do prefer the newer format too, maybe it's worth enforcing a max line length with `lll` to re-format too-long-lines everywhere in our code).

```suggestion
				require.NoErrorf(t, err, "rlp.DecodeBytes(rlp.EncodeToBytes(%T), %T) resulted in %s",
					(*withoutMethods)(body), got, pretty.Sprint(got))
```

Reply:
It was missed in #130 because of auto-merge so I added it as the very first thing I did. Yes it's out absolutely of scope but it's also not worth reverting here only to create a new PR for the sake of purity.